project/accounts/models.py

```diff
@@ -169,10 +169,10 @@ def resize_profile_image(self):
         """
         Resizes and crops the user uploaded image and creates a thumbnail version of it
         """
-        profile_image_field = self.profile_image
-        image_file = io.StringIO(profile_image_field.read())
-        profile_image = Image.open(image_file)
-        profile_image.load()
+        #profile_image_field = self.profile_image
```

Comment:
It is not a good practice to store unused code lines as commented. You can remove them.

Reply:
As suggested, remove unused code.


```suggestion
```